go-controller/pkg/podannotation/external_gw_conntrack.go

```diff
@@ -1,7 +1,7 @@
 //go:build linux
 // +build linux
 
-package util
+package podannotation
```

Comment:
isn't it weird to put this external gw conntrack code under podannotation? I don't see what it has to do with it.

Reply:
yeah so the problem is that it is the only file that was using podannotation within util package, so I just moved it together. It probably should go to its own package, but even better would be to get rid of the old egressgw implementation and then cleanup.
I guess just moving it to its own package is the best I can do for now